jsonnet/kube-prometheus/alertmanager/alertmanager.libsonnet

```diff
@@ -109,7 +109,8 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
           name: 'alertmanager',
           namespace: $._config.namespace,
           labels: {
-            'k8s-app': 'alertmanager',
+            'app.kubernetes.io/name': 'alertmanager',
```

Comment:
do the other alertmanager objects have these labels already?

Reply:
Updated the PR to include the labels